blank-canvas-blocks/sass/blocks/_pullquote.scss

```diff
@@ -26,3 +25,10 @@
 		color: var(--wp--custom--color--background);
 	}
 }
+
+.wp-block-pullquote:not(.is-style-solid-color) {
```

Comment:
Is that how we would expect Gutenberg to deal with the scenario?  It feels theme-specific to me and thus no business of BCB's.  Another theme might have a preference for the blockquote paddings to be the same for both and there would be no way to do that.

Reply:
These rules are already in BCB, this change just makes them less powerful. I'm happy to remove them from BCB but we need to check the impact on the other child themes if we do.